examples/reprsimil/brsa_representational_similarity_estimate_example.ipynb

```diff
@@ -7,7 +7,7 @@
    },
    "source": [
     "# This demo shows how to use the Bayesian Representational Similarity Analysis method in brainiak with a simulated dataset.\n",
-    "Questions can be directed to mcai [ at ] princeton [ dot ] edu"
+    "*Feedbacks and questions are welcome. Please direct them to mcai [ at ] princeton [ dot ] edu"
```

Comment:
Not really what the change is about, but do you mind removing this line? We are already instructing users to open GitHub issues for BrainIAK or contact us on Gitter.